src/deploy/dataconnect/prepare.ts

```diff
@@ -55,6 +58,36 @@ export default async function (context: any, options: Options): Promise<void> {
     filters,
   };
   utils.logLabeledBullet("dataconnect", `Successfully prepared schema and connectors`);
+  if (options.dryRun) {
```

Comment:
From chatting offline -- we should invoke `diffSchema` here to surface migration diffs in `dryRun`